tests/func/test_checkout.py

```diff
@@ -850,3 +850,24 @@ def test_checkouts_for_pipeline_tracked_outs(tmp_dir, dvc, scm, run_copy):
 
     (tmp_dir / "ipsum").unlink()
     assert set(dvc.checkout()["added"]) == {"bar", "ipsum"}
+
+
+def test_checkout_external_modify(tmp_dir, dvc, scm, s3):
```

Comment:
```suggestion
@pytest.mark.parametrize("workspace", pytest.lazy_fixture("s3"), indirect=True)
def test_checkout_external_modify(tmp_dir, dvc, scm, workspace):
```

Reply:
This will setup cache for you, and you can then use `workspace` object as remote tmp_dir